python/cudf/cudf/tests/test_serialize.py

```diff
@@ -35,11 +41,47 @@
             {"x": ["a", "bb", "ccc"], "y": [1.0, None, 3.0]},
             index=[1, None, 3],
         ),
-        pd._testing.makeTimeDataFrame,
+        pd._testing.makeBoolIndex,
+        pd._testing.makeCategoricalIndex,
+        lambda: pd._testing.makeCustomDataframe(3, 4),
+        lambda: pd._testing.makeCustomIndex(2, 5),
+        pd._testing.makeDataFrame,
+        pd._testing.makeDateIndex,
+        pd._testing.makeFloatIndex,
+        pd._testing.makeFloatSeries,
+        pd._testing.makeIntIndex,
+        pd._testing.makeIntervalIndex,
+        pd._testing.makeMissingDataframe,
         pd._testing.makeMixedDataFrame,
+        pd._testing.makeMultiIndex,
+        lambda: pd._testing.makeNumericIndex(dtype=np.float64),
+        pd._testing.makeObjectSeries,
+        pytest.param(
+            pd._testing.makePeriodFrame,
+            marks=pytest.mark.xfail(
+                reason="Periods not supported in cudf", raises=RuntimeError
+            ),
+        ),
+        pytest.param(
+            pd._testing.makePeriodIndex,
+            marks=pytest.mark.xfail(
+                reason="Periods not supported in cudf", raises=RuntimeError
+            ),
+        ),
+        pytest.param(
+            pd._testing.makePeriodSeries,
+            marks=pytest.mark.xfail(
+                reason="Periods not supported in cudf", raises=RuntimeError
+            ),
+        ),
+        pd._testing.makeRangeIndex,
+        pd._testing.makeStringIndex,
+        pd._testing.makeStringSeries,
         pd._testing.makeTimeDataFrame,
-        # pd._testing.makeMissingDataframe, # Problem in distributed
-        # pd._testing.makeMultiIndex, # Indices not serialized on device
```

Comment:
These it turns out work now, and almost all of the others in `pd._testing` do as well, so I just added them (this exposed #10785).